index.bs

```diff
@@ -166,11 +166,12 @@ Requirements {#requirements}
 This section outlines functional and non-functional requirements of HTTPS/WSS
 usage in local network represented in [[#usecases]].
 
-Functional requirements consist of device discovery, device identification,
-certificate issuance, and certificate management and lifecycle.
+Functional requirements consist of guarantee of device trustworthiness,
+device identification, alerting user and user’s approval, device discovery,
+and certificate management.
 
-Non-functional requirements address user experience and security aspects of
-HTTPS/WSS usage in local network.
+Non-functional requirements address security, privacy, availability, scalability,
+and user experience aspects of HTTPS/WSS usage in local network.
 
 Issue: Currently this document only describes the requirements for the use
 cases where the UA and the device communicates within the local network.
```

Comment:
Why not remove this issue? Now your PR has added mentions about "web service". It looks like the issue has been resolved by your PR, right?

Reply:
Although the web service always communicate with the device through the UA in the local network, I have mentioned the device that has a Web PKI certificate. So we can remove the Issue.